crates/futures/src/lib.rs

```diff
@@ -32,6 +32,7 @@
 
 #![cfg_attr(target_feature = "atomics", feature(stdarch_wasm_atomic_wait))]
 #![deny(missing_docs)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
This should be covered in the CI as well.
https://github.com/rustwasm/wasm-bindgen/blob/9aa86a6ecbcc5d6f47e335f7bcae3e2b62ff852a/.github/workflows/main.yml#L420-L435.

Reply:
Ah yes good spot.